mysqld_exporter.go

```diff
@@ -349,79 +349,79 @@ var (
 				"The number of concurrent connections for this user.",
 				[]string{"user"}, nil)},
 		"CONNECTED_TIME": {prometheus.CounterValue,
-			prometheus.NewDesc(prometheus.BuildFQName(namespace, informationSchema, "user_statistics_connected_time"),
+			prometheus.NewDesc(prometheus.BuildFQName(namespace, informationSchema, "user_statistics_connected_time_total"),
```

Comment:
Oh right, same for the metrics below, they also need a time unit. I don't know it, hopefully @SuperQ does.